<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="textEdit_recv">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>521</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_gga">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>211</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_info">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>400</y>
     <width>301</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clear">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>490</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="Serial">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>20</y>
     <width>161</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lsport">
      <property name="text">
       <string>Portl</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QComboBox" name="cbsport">
      <property name="editable">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>COM20</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lsbaud">
      <property name="text">
       <string>Baud</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QComboBox" name="cbsbaud">
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <item>
       <property name="text">
        <string>4800</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>115200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>230400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>460800</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lsdata">
      <property name="text">
       <string>Databit</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QComboBox" name="cbsdata">
      <property name="editable">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="lsparity">
      <property name="text">
       <string>Parity</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QComboBox" name="cbsparity">
      <property name="editable">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>N</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>E</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>O</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="lsstop">
      <property name="text">
       <string>Stopbit</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QComboBox" name="cbsstop">
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <item>
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_open">
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="pushButton_refresh">
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="Ntrip">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>210</y>
     <width>161</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="lncaster">
      <property name="text">
       <string>Caster</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_caster"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lnport">
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_port"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lnmount">
      <property name="text">
       <string>MountP</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_mount"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="lnuser">
      <property name="text">
       <string>User</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_user"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="lnpwd">
      <property name="text">
       <string>PassWd</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_pwd"/>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="pushButton_conn">
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="ATCmd">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>390</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="label_atcmd">
      <property name="text">
       <string>AT Cmd</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton_atcmd">
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
